sklearn/ensemble/_bagging.py

```diff
@@ -126,7 +123,11 @@ def _parallel_build_estimators(
         )
 
         # Draw samples, using sample weights, and then fit
+        support_sample_weight = has_fit_parameter(
+            ensemble.base_estimator_, "sample_weight"
+        )
         if support_sample_weight:
+            sample_weight = fit_params.get("sample_weight")
             if sample_weight is None:
```

Comment:
It seems that this is something that should use `sklearn.utils.validation._check_sample_weight`.

Reply:
We check sample weight earlier: https://github.com/BenjaminBossan/scikit-learn/blob/1221996815589ef076f9c2ac7434ea28a9dcf6bc/sklearn/ensemble/_bagging.py#L395

I think there is no advantage of checking again at the suggested line.

Reply:
It will create for you the array of `ones` and trigger a copy if requested.

Reply:
Done